extension.ts

```diff
@@ -346,11 +362,14 @@ const findVSCodeRecentItems = (
 
     const storage = JSON.parse(ByteArray.toString(contents));
 
-    const recentWorkspaceURIs: string[] =
+    const recentWorkspaceItems: (string | { configURIPath: string })[] =
```

Comment:
While you're at this please also change the type to `ReadonlyArray<…>`.

Reply:
:+1: